assets/src/edit-story/elements/gif/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { PanelTypes } from '../../components/panels';
+import { SHARED_DEFAULT_ATTRIBUTES } from '../shared';
+import { MEDIA_DEFAULT_ATTRIBUTES, MEDIA_PANELS } from '../media';
+export { default as Display } from '../media/imageDisplay';
+export { default as Edit } from '../media/imageEdit';
```

Comment:
I don't think this file (`imageEdit`) should have been moved this way.

`ImageEdit` and `VideoEdit` are just using `MediaEdit` but with different prop types - so could `GifEdit`.

Reply:
Same with `imageFrame`

Reply:
Good point, I'll make this change :)